cfme/tests/automate/test_customization_paginator.py

```diff
@@ -82,7 +92,7 @@ def test_paginator(some_dialogs, soft_assert, appliance):
         current_rec_offset = view.paginator.min_item
         current_rec_end = view.paginator.max_item
 
-        assert current_rec_offset <= current_rec_end <= current_total, \
+        assert int(current_rec_offset) <= int(current_rec_end) <= int(current_total), \
```

Comment:
:+1:

Reply:
Should we not switch these out into the paginator properties?

Reply:
Will let the author of this file know after this PR :)